### DISCUSSION TITLE
failed finding central directory

### DISCUSSION THREAD

**Discussion by EatonL** [open-mmlab/mmdetection#8730] [Q&A] [answered]
when i resume from my checkpoint trained before,it report a error:"RuntimeError: [enforce fail at inline_container.cc:144] . PytorchStreamReader failed reading zip archive: failed finding central directory"

here is my configuration：
pytorch 1.6.0
mmcv-full 1.2.7

what can i do to solve this problem?THKS

**Comment by hhaAndroid** [comment, marked as answer]
Maybe you should check the pth is correct.

**Comment by HristoBuyukliev** [comment]
I confirm the same problem. I've been training CentripetalNet on a custom dataset; It trains successfully, but if I try to resume from previous checkpoint, it gives the same error.

     python train.py custom_configs/centripetal/default.py --resume-from /root/sasha_code/checkpoints/centripetal/epoch_35.pth

Notice how I'm giving absolute path; I am also certain the .pth file is there.

**Comment by qiangliuhfut** [comment]
我也出现这个问题，   后来发现是我得epoch_20.pth没有上传完整

**Comment by Matthew1204** [comment]
Hello, I have a similar problem. I interrupted when downloading the weight file, so I couldn't find the weight file. How can I find the location of the interrupted download file and delete it?

**Comment by YangXiaoYang1997** [comment]
> Hello, I have a similar problem. I interrupted when downloading the weight file, so I couldn't find the weight file. How can I find the location of the interrupted download file and delete it?

in home/.cache/xxxxx/.../checkpoints
